### PR TITLE
feat: update download info for 0.12.0 release

### PR DESCRIPTION
**Pull request by zyxxoo**

### CODE REVIEW

File: download.md [hunk 1]
```diff
@@ -2,20 +2,21 @@
...
-The latest HugeGraph: **0.11.2**, released on _2019-11-20_.
+The latest HugeGraph: **0.12.0**, released on _2022-12-24_.
 
 components       | description          | download
 ---------------- | -------------------- | ----------------------------------------------------------------------------------------------------------------
-HugeGraph-Server | HugeGraph的主程序      | [0.11.2](https://github.com/hugegraph/hugegraph/releases/download/v0.11.2/hugegraph-0.11.2.tar.gz)
+HugeGraph-Server | HugeGraph的主程序      | [0.12.0](https://github.com/hugegraph/hugegraph/releases/download/v0.12.0/hugegraph-0.12.0.tar.gz)
 HugeGraph-Hubble | 基于Web的可视化图形界面  | [1.5.0](https://github.com/hugegraph/hugegraph-hubble/releases/download/v1.5.0/hugegraph-hubble-1.5.0.tar.gz)
-HugeGraph-Loader | 数据导入工具            | [0.11.1](https://github.com/hugegraph/hugegraph-loader/releases/download/v0.11.1/hugegraph-loader-0.11.1.tar.gz)
+HugeGraph-Loader | 数据导入工具            | [0.11.3](https://github.com/hugegraph/hugegraph-loader/releases/download/v0.11.3/hugegraph-loader-0.11.3.tar.gz)
```
Comment (javeme) [inline download.md:11]: can also keep hugegraph-loader-0.12.0.tar.gz

File: download.md [hunk 2]
```diff
@@ -2,20 +2,21 @@
...
+[0.12.0](https://github.com/hugegraph/hugegraph/releases/download/v0.12.0/hugegraph-0.12.0.tar.gz)  | [1.9.6](https://mvnrepository.com/artifact/com.baidu.hugegraph/hugegraph-client/1.9.6)  | [0.11.3](https://github.com/hugegraph/hugegraph-loader/releases/download/v0.11.1/hugegraph-loader-0.11.3.tar.gz)   | [0.11.0](https://github.com/hugegraph/hugegraph-studio/releases/download/v0.11.0/hugegraph-studio-0.11.0.tar.gz) &#124; [1.5.0](https://github.com/hugegraph/hugegraph-hubble/releases/download/v1.5.0/hugegraph-hubble-1.5.0.tar.gz)      |       | [2.0.1](https://mvnrepository.com/artifact/com.baidu.hugegraph/hugegraph-common/2.0.1)  | [1.5.0](https://github.com/hugegraph/hugegraph-tools/releases/download/v1.5.0/hugegraph-tools-1.5.0.tar.gz)
```
Comment (javeme) [inline download.md:19]: hugegraph-client version can be set to 2.0.0
hugegraph-loader version can be set to 0.12.0
hugegraph-tools version can be set to 1.6.0
hugegraph-hubble version can be set to 1.6.0

File: download.md [hunk 3]
```diff
@@ -2,20 +2,21 @@
...
-HugeGraph-Hubble | 基于Web的可视化图形界面  | [1.5.0](https://github.com/hugegraph/hugegraph-hubble/releases/download/v1.5.0/hugegraph-hubble-1.5.0.tar.gz)
-HugeGraph-Loader | 数据导入工具            | [0.11.1](https://github.com/hugegraph/hugegraph-loader/releases/download/v0.11.1/hugegraph-loader-0.11.1.tar.gz)
-HugeGraph-Tools  | 命令行工具集            | [1.5.0](https://github.com/hugegraph/hugegraph-tools/releases/download/v1.5.0/hugegraph-tools-1.5.0.tar.gz)
+HugeGraph-Server | HugeGraph的主程序      | [0.12.0](https://github.com/hugegraph/hugegraph/releases/download/v0.12.0/hugegraph-0.12.0.tar.gz)
+HugeGraph-Hubble | 基于Web的可视化图形界面  | [1.6.0](https://github.com/hugegraph/hugegraph-hubble/releases/download/v1.6.0/hugegraph-hubble-1.6.0.tar.gz)
+HugeGraph-Loader | 数据导入工具            | [0.12.0](https://github.com/hugegraph/hugegraph-loader/releases/download/v0.12.0/hugegraph-loader-0.12.0.tar.gz)
+HugeGraph-Tools  | 命令行工具集            | [1.6.0](https://github.com/hugegraph/hugegraph-tools/releases/download/v1.6.0/hugegraph-tools-1.6.0.tar.gz)
 HugeGraph-Spark  | 基于Spark GraphX的图分析工具 |
```
Comment (imbajin) [inline download.md:13]: @javeme  spark 还留着么? 似乎去掉好点,  如果需要可以先放个 computer (标注 BETA 之类的)

Comment (javeme) [reply download.md:13]: +1

**Review by imbajin — DISMISSED** [review]

File: download.md [hunk 4]
```diff
@@ -2,33 +2,34 @@
...
+HugeGraph-Loader | 数据导入工具            | [0.12.0](https://github.com/hugegraph/hugegraph-loader/releases/download/v0.12.0/hugegraph-loader-0.12.0.tar.gz)
+HugeGraph-Tools  | 命令行工具集            | [1.6.0](https://github.com/hugegraph/hugegraph-tools/releases/download/v1.6.0/hugegraph-tools-1.6.0.tar.gz)
 
 ### Versions mapping
 
-server                                                                                           | client | loader                                                                                                                                                                      | studio &#124; hubble                                                                                                             | spark | common | tools
+server                                                                                           | client | loader                                                                                                                                                                      | studio | hubble                                                                                                             | common | tools
```
Comment (imbajin) [inline download.md:16]: we should also remove the "studio" column (due to archive it)

Comment (zyxxoo) [reply download.md:16]: i have deleted, maybe refresh can see the newest commit;

File: download.md (hunk 3 above)
Comment (zyxxoo) [reply download.md]: I have deleted

File: download.md (hunk 4 above)
Comment (zyxxoo) [reply download.md:16]: ok, but the history version has studio, maybe the old version needs a studio?

Comment (imbajin) [reply download.md:16]: yep, consider do like this, move it to the last column, and set the `/` after 0.10
image

**Review by imbajin — DISMISSED** [review]

File: download.md [hunk 5]
```diff
@@ -2,33 +2,34 @@
...
+[0.12.0](https://github.com/hugegraph/hugegraph/releases/download/v0.12.0/hugegraph-0.12.0.tar.gz)  | [2.0.1](https://mvnrepository.com/artifact/com.baidu.hugegraph/hugegraph-client/2.0.1)  | [0.12.0](https://github.com/hugegraph/hugegraph-loader/releases/download/v0.12.0/hugegraph-loader-0.12.0.tar.gz)   | [1.6.0](https://github.com/hugegraph/hugegraph-hubble/releases/download/v1.6.0/hugegraph-hubble-1.6.0.tar.gz)  &#124; [0.12.0](https://github.com/hugegraph/hugegraph-studio/releases/download/v0.12.0/hugegraph-studio-0.12.0.tar.gz)       | [2.0.1](https://mvnrepository.com/artifact/com.baidu.hugegraph/hugegraph-common/2.0.1)  | [1.6.0](https://github.com/hugegraph/hugegraph-tools/releases/download/v1.5.0/hugegraph-tools-1.6.0.tar.gz)
```
Comment (imbajin) [inline download.md:18]: consider remove studio v0.12 due to apache version will not include it (we won't adopt it in v0.12, only keep hubble)

image

**Review by javeme — APPROVED** [review]

**Review by zhoney — APPROVED** [review]

**Pull request merged**